packages/zoltan2/test/core/partition/MultiJaggedTest.cpp

```diff
@@ -1412,15 +1412,19 @@ int main(int narg, char *arg[])
     bool test_boxes = false;
     bool rectilinear = false;
 
-#ifdef KOKKOS_ENABLE_CUDA
+#if defined(KOKKOS_ENABLE_CUDA)
     // make a new node type so we can run BasicVectorAdapter with UVM off
     // The Tpetra MV will still run with UVM on and we'll compare the results.
     // For Serial/OpenMP the 2nd test will be turned off at the CMake level.
     // For CUDA we control uvm on/off with parameter uvm set to 0 or 1.
     // TODO: Probably this should all change eventually so we don't have a node
     // declared like this.
-    typedef Kokkos::Compat::KokkosDeviceWrapperNode<
-      Kokkos::Cuda, Kokkos::CudaSpace>  uvm_off_node_t;
+    using uvm_off_node_t = Kokkos::Compat::KokkosDeviceWrapperNode<
+      Kokkos::Cuda, Kokkos::CudaSpace>;
+#endif
+#if defined(KOKKOS_ENABLE_HIP)
```

Comment:
Should this be an `elif`, like above?

Reply:
Not necessarily, the `elif` was necessary in Zoltan2_TaskMapping.hpp but since there is no `else` close here it does not need to be.